experimental/dds/tree2/src/test/simple-tree/utils.ts

```diff
@@ -36,6 +54,33 @@ export function getRoot<TRoot extends TreeFieldSchema>(
 	return view.root;
 }
 
+/**
+ * Helper for making small test schemas.
+ */
+export function makeSchema<TSchema extends ImplicitFieldSchema>(
```

Comment:
since schema factory is effectively stateless (only state is for deduplication of structural schema), I think it would be reasonable for each test file to simple do `const schemaFactory = new SchemaFactory("test")` at the top of the file (or in tests if more appropriate) instead of importing and using this. Unless we want to be able to use multiple schema with the same name in the same test, making different scopes should be unneeded.